backend/category_routes.py

```diff
@@ -16,9 +16,6 @@ def create_category(
     db: Session = Depends(get_db),
     current_user: User = Depends(get_current_user)
 ):
-    """
-    Create a new category
-    """
```

Comment:
Let's leave these please they are docs. We should enrich them to describe the functions instead.